[PWR-272] skill suggestions are now sorted.

ntenelsen opened the pull request:
Danke nochmal für die rebase commands 👍 

(uuuuund mir ist gerade aufgefallen, dass es für die Sortierung hier ja gar keinen Test gibt ._.)

ntenelsen committed:
[PWR-272] skill suggestions are now sorted.

Niklas345 commented on src/main/java/de/hbt/pwr/profile/service/SkillRecommendationService.java:
```diff
@@ -54,6 +61,7 @@
                 .flatMap(Collection::stream)
                 .filter(s -> (!project.getSkills().contains(s)))
                 .distinct()
+                .sorted(new SkillNameComparator())
```
```suggestion
                .sorted(Comparator.comparing(Skill::getName))
```
Spart die die anonyme Klasse und ist genau dafür gemacht

Niklas345 commented on src/main/java/de/hbt/pwr/profile/model/Skill.java:
```diff
@@ -14,6 +11,7 @@
 @AllArgsConstructor
 @NoArgsConstructor
 @Data
+@EqualsAndHashCode
```
Nimm doch vielleicht noch die Versionen aus der EqualsAndHashcode raus. Wenn du schon dabei bist, wirf ne @ToString mit dran!

Niklas345 reviewed (changes requested):
Kleinigkeit!

ntenelsen committed:
[PWR-272] Implemented change requests.
Replaced anonymous Comparator class and adjusted equals implementation of the Skill class.
